### PR TITLE
New package: libomp-7.0.1

### PR DESCRIPTION
**Pull request by atweiden**

### CODE REVIEW

File: srcpkgs/libomp/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+version=7.0.0
+revision=1
+build_style=cmake
+configure_args="-DLIBOMP_ENABLE_SHARED=ON -G Ninja"
+make_cmd="ninja"
+hostmakedepends="ninja perl pkg-config"
+makedepends="elfutils-devel libffi-devel"
+short_desc="LLVM OpenMP Runtime Library"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="Custom"
```
Comment (maxice8) [inline srcpkgs/libomp/template]: Off the top of my head i think all LLVM projects are under the NCSA license

File: srcpkgs/libomp/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+do_install() {
+	cd ${cmake_builddir:=build}
+	DESTDIR=${DESTDIR} ninja install
+}
+
+post_install() {
+	vlicense LICENSE.txt
+	# libgomp.so is also provided by libgomp; remove it to fix the conflict
+	rm -f ${DESTDIR}/usr/lib/libgomp.so
```
Comment (maxice8) [inline srcpkgs/libomp/template]: Should problably just be included in the -devel package.

libgomp is provided by libgomp-devel, make them conflict ?

File: srcpkgs/libomp/template [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+make_cmd="ninja"
+hostmakedepends="ninja perl pkg-config"
+makedepends="elfutils-devel libffi-devel"
+short_desc="LLVM OpenMP Runtime Library"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="Custom"
+homepage="https://openmp.llvm.org"
+distfiles="https://releases.llvm.org/$version/openmp-$version.src.tar.xz"
+checksum=30662b632f5556c59ee9215c1309f61de50b3ea8e89dcc28ba9a9494bba238ff
+wrksrc="openmp-$version.src"
```
Comment (maxice8) [inline srcpkgs/libomp/template]: please put wrksrc directly  below revision=

**Comment by atweiden** [comment]
ok

**Comment by maxice8** [comment]
After all.

shouldn't this be part of the llvm package ?

**Comment by atweiden** [comment]
@maxice8 I was mimicing the way it works on macOS. If it should be made into a subpackage, that's ok by me. Can you take it over then? Note this pkg is intended for use with cryfs-0.10, whenever it gets released...

**Pull request closed without merging**